backend/sftp/options_test.go

```diff
@@ -498,158 +500,73 @@ func setupKeyFiles(tmpdir string) (*keyFiles, error) {
-I/Zsoo9jXT18WgahDFbtUMwCgzEFXHZ2MwAbfMzYKF1ECt0fszSro8WL8uVPrFaB
-vbqVJ4linsBDjLdXbxHcKV1gZalKjcukWdJapQBAYxsE21WrNSL+nsW7mw8XXI1f
-40jy0KA+2qyp2I1MxdCrZP7KDFdYe6ayzeKomoLY8wAq25iWf5XJAh1fkm/guF9w
-+Lq9hRWkM5//LZ/vcvmyJuuMABj0PTXeUqGlCc4YVuEnOHaN1m0CggEBAP7cYR44
-9tQupzmJwXYJnNI5R/16ES0njJRmUkCdYUlGBwJ++4dJpD3GGDkgCotc5AzD5QWF
-Z4VI7qK1vUlP7g9M3QEbOrqyZVBfHGKvaqXSXf6gg78Ex8Dt9ip/DsbJn//t0cnj
-zQ5eFLh8Vl4Gz/QhRAUmCbXXl8YV9vUnzAVKe2eiF8iZOWALqB17H3rZahxsnmZt
-Scm++NzlxZf2CdcC0/gq06YGHKMaCEJAuoEOSy5GmC0YpCNe5fcpTNpg+VUlomc+
-NIgiQ3tg/SUpPc/q5wERoDLZpHezREwyNalx8kfkuhIbek4oS3W8Vibm3JjbpF/M
-roHqBjs+j2jf/08CggEBANH/oiHMR+U+eBCAQZu72WR3gnKNCNSqHmjY957/f0QN
-a13RrnoHOIeqhMt35K/HIgYLdVMA4EimUTRFnq9SDeqgfb3tWtVVB1lT1ZS71cZI
-6HWhCT/m/AABmJCIWGqNFVePbUCY6TyDowsMLkjqOzJ104eO2Y+oZOOHlEloElbO
-qArep41HDRIUrXOio6H4ZzjJlSDMgBeU4DsL4SitK459v4u/osScwf5ZyHNAGmqB
-Rs7r9NcIbYpUueaQVVdVf95AxK1ZQ7msZKW3YjyJz8Z9kjsdcTRMETbOxEsB/03U
-7Q5rQMr37uZNxbINnX2fh5hn9XVarPS+SDwNGME6pEcCggEBALryWwb5UA16n020
-f8We7XrDa8xCYyEVNqiQmdst1nQSOwgYr1agrSpnCdO1biamH95BP9iZ78K0KeAO
-oeeKCx0MC71JBP5355tZ+Q9mjztNoYcqpRlUX1Zk90Ja6zLkKUppX47RW9QjLN3a
-ztuv8ZCpaiTArzTFDV7PM9TGuYBUD0uIehu6UXzjcBEYBJJvssdg4ZxOpGapgBFB
-NnzujG88ctJCT/gj2ZPGf7Jhmq0aGAm83NmPjq8naFax4974bUyJC6Th21TUlV2G
-WoqMwvul2odNL469WUg4pmuiFPzTSZ506AxqPX/hTODzItrsU3qI+v0Ovh8r1CBX
-Fokebj8CggEAWiifilUzNNgKIkN+Z4dSAVFR/y5P8UYMgkVMosXc9PGx+/ivKRL6
-kTyDgPu7gkBDekbnGzjQEkDdskyFoY3gDbDT63wBOIAmBJL6qr2uPVBGBWKbHwVj
-gfktcDgpha2G0S3x4P8FfAakNHUJViLCQZrWs2eAPq40in9GCfIVlZFqEiif1QcB
-NJcOFQxppnuIjZf2X7uM7xLq5k7mX1lhzu5sE2q2TiVjIHmZlumZrcpNBT/GwZ+L
-sA1KNxQWn8VEfb5e8nHVotzB5WgDVCxyuSxmYNz2IlbaOSayneWAoADfugYQLlQe
-DGCtlRFFYY7hX2yatMS2ZulfB/EzhJpRtwKCAQEAnyFVAnBYc2yuXQbuQ2OBMKEr
-AUAl/Ti8Oexv16SLDCTfYhZ0IgRwrtWhsHDIw5b5mbuaRqkEfPi1TyoCiEXwsKMb
-uQVn7xxPzvM2CuUXWN8Kl3HCmkx1lnHxQbBUoynd9LCeVgw6Nl50QAjeMJgKh/c+
-vqjW2lqen/rHe68l/x4fWrpP3X+C8gtlHys4FtE1IVlAIorhAGgHTddxqq/YA0fy
-kN/MZe2ffwIEfnV+tzpVo/EIxOmiL8qOqe/ulXx/E7bDtTST+bBenfu1Zj01u7J/
-+nMg5AAaDs3iPpgQGJDGeJP1s1ib2SVVcasdlhzVhESAeaFKE5i6AZg4RN3xNw==
------END RSA PRIVATE KEY-----` // #nosec - these are generated specifically for testing
-
-const keyWithoutPassphrasePubkey = `ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDREGo3hNkWxVOjMml8lm8bjIpwqG+daIZ8/gO+T9RxTJrQ243WoA9qn9hl9zabTa6QMXFiIYY+RIut+MQO4+kLIgAPxGFedbKi/aZdLvTiKX5MJGrKvnz1V0I/b85+TpacCn0rdkYAa0b1zJy9DVwFF54KQ4r5KmO/o1Flz69aY9PV6Kf7Gnj7JardgQjxGx7n01PlQgAh7AWLImhR5RfJw7r1q2ecuVkg3qoe4aKn3KEXafP6YWYepfKSRKtJ23NIAFgzKVs/Ub/uG2/DilJeZQe9tlREB22us2X+5bM7jw/oy1tcIG9/L9SNMaVRVhP7gZPKWtqiXmj8mvjOP09LxYHu9PoYIC0MN6hCnxrGbD+Z+dPYpk4NiDThauSdKp6dOTH3ZRMhgshQIONLLTv2q+FACRKjBVSUNeqKyksFv0vcZuU0HBTjAbTRbzdx6A9Cle71E4nACw4qDA6nL9p3YYHq2oqCNMH4kq93y6IEO4kKIpQlM1xtTL9OMSkIhC2qT9QLU1vUeIDxWg0b0ikXrFCpk3wJqB/H4kIUOYWyTzmEAz4rLFR+IuvGfPE7fUEJiwcl/EMAbkr3FsEODcbf0+RAkA0r9WKAqmWKRHXozg1JwEbRMHOMFiCrKuTxm6n66vHoZykRJPUeVyyW0R9RXByzqqjZijE8i82X/JRq6Q==` // nolint:gosec,lll // these are generated specifically for testing
```

Comment:
While these keys gave no access to anything anywhere, the sftp authentication opts tests need a valid keypair to run.  Unfortunately, this causes security-related alerts about having credentials hard-coded.  So now we simply generate the test keypair.